public/app/features/dashboard/components/TransformationsEditor/TransformationsEditor.tsx

```diff
@@ -359,7 +359,9 @@ class UnThemedTransformationsEditor extends React.PureComponent<TransformationsE
           search={search}
           suffix={suffix}
           xforms={xforms}
-          setState={this.setState.bind(this)}
```

Comment:
I removed this use of the parent components `setState` in favor of normal callbacks. I didn't like that the parent needed to implement a very specific state signature.